model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -232,17 +234,34 @@ object Workspace {
       None,
       None,
       Option(createdDate),
-      workspaceType = WorkspaceType.RawlsWorkspace
+      workspaceType = WorkspaceType.RawlsWorkspace,
+      state = WorkspaceState.Ready
     )
   }
-
+  def buildReadyMcWorkspace(namespace: String,
```

Comment:
Renamed to make it clear that this puts the workspace into Ready state, and thus should only be used by tests.